app/src/main/res/values/strings.xml

```diff
@@ -66,26 +66,26 @@
     <string name="pref_about_privacy_title">Privacy</string>
     <string name="pref_about_license" translatable="false">pref_about_license</string>
     <string name="pref_about_license_title">License</string>
-    <string name="pref_about_license_summary">GNU General Public License v3.0</string>
+    <string name="pref_about_license_summary">GNU General Public License v3.0+.</string>
     <string name="pref_about_version" translatable="false">pref_about_version</string>
     <string name="pref_about_version_title">Version</string>
 
 
     <!-- CALENDAR SYNC //-->
-    <string name="calendar_not_responded_title">Events I have not Responded To</string>
-    <string name="calendar_declined_title">Events I\'m not Attending</string>
-    <string name="calendar_tentative_title">Events I May Be Attending</string>
-    <string name="calendar_attending_title">Events I\'m Attending</string>
+    <string name="calendar_not_responded_title">Unanswered</string>
```

Comment:
These are the actual calendar names as displayed in the phone calendar, which I'm not sure if I want to change at this point (although I'm grateful that you came up with shorter names :-) ).

There's a plan for one of the next releases to allow users to customize the calendar names, so I think that will be a good point to rename the calendars, but for now let's keep the originals please.